Dockerfile.pip

```diff
@@ -1,33 +1,35 @@
 # This Dockerfile creates an environment suitable for downstream usage of AllenNLP.
 # It creates an environment that includes a pip installation of allennlp.
 
-FROM python:3.6.8-stretch
+FROM nvidia/cuda:10.0-base-ubuntu18.04
 
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-ENV PATH /usr/local/nvidia/bin/:$PATH
-ENV LD_LIBRARY_PATH /usr/local/nvidia/lib:/usr/local/nvidia/lib64
-
-# Tell nvidia-docker the driver spec that we need as well as to
-# use all available devices, which are mounted at /usr/local/nvidia.
-# The LABEL supports an older version of nvidia-docker, the env
-# variables a newer one.
-ENV NVIDIA_VISIBLE_DEVICES all
-ENV NVIDIA_DRIVER_CAPABILITIES compute,utility
-LABEL com.nvidia.volumes.needed="nvidia_driver"
-
 WORKDIR /stage/allennlp
 
 ARG VERSION
 ARG SOURCE_COMMIT
 
+RUN apt-get update && apt-get install -y --no-install-recommends \
+         build-essential \
+         cmake \
+         git \
+         python3-dev \
```

Comment:
can we be more specific about what version of python to install? off the top of my head I have no idea what version this would result in

Reply:
I'd have to use Conda.  I previously looked into that.

```
# python --version
Python 3.6.8
```

Reply:
I pushed a commit that forces 3.6 at least.